graphql/schemabuilder/types.go

```diff
@@ -85,6 +90,9 @@ func (s *Object) Key(f string) {
 type method struct {
 	MarkedNonNullable bool
 	Fn                interface{}
+
+	// Connection configuration
```

Comment:
?  Is this meant as a header for the section?

Reply:
Yeah - I can remove it for now if it's confusing